compiler/src/iree/compiler/API/MLIRInterop.h

```diff
@@ -28,6 +28,11 @@ ireeCompilerRegisterDialects(MlirDialectRegistry registry);
 
 // Gets the MlirContext that the session manages. The context is owned by the
 // session and valid until it is destroyed.
+// This implicitly "activates" the session, consuming command line options and
```

Comment:
Does this reconsume them? Or why would borrow consume these?

Reply:
Can say it differently: Basically, if an outside actor is requesting access to the context, it will force the session to initialize the context, which may involve processing some command line options. In other words, set command line options before getting the context or doing other things that cause lazy initialization.